### ISSUE TITLE
improve jpeg size estimate

### ISSUE THREAD

**Issue by jepler** [adafruit/Adafruit_CircuitPython_OV5640#26]
The jpeg size estimate in `capture_buffer_size` is causing folks trouble:
```
        if self.colorspace == OV5640_COLOR_JPEG:
            return self.width * self.height // self.quality
```
replace it with a better heuristic after gathering information.

note to users affected by this: Instead of using `capture_buffer_size` in jpeg mode, manually determine the size for your quality & resolution settings. When this value is too small, the `capture` routine returns None in jpeg mode.

**Comment by jepler** [comment]
espressif's esp32-camera uses `width * height / 5` regardless of quality level.

**Comment by jmangum** [comment]
Confirming that setting the size of the capture buffer to `cam.capture_buffer_size` does for many settings of `cam.quality` truncate the `buf = bytearray(cam.capture_buffer_size)` such that the end-of-image (EOI) bit is missing, and `cam.capture(buf)` returns `None` rather than a `memoryview` jpeg.  As noted, key is to set the size of `buf` large enough.  After quite a bit of experimentation, where I am capturing images of a wide range of scenery, so far it appears that using esperssif's esp32-camera value of `cam.width*cam.height/5` (perhaps this is supposed to be a floor divide, or `//5`?) works well.  Will deploy my remote camera system with this setting and see how it goes.

**Comment by jepler** [comment]
(the quoted code was "C" code in which "/" behaves similar to Python floor division "//" when the arguments are of integer type)

**Comment by jmangum** [comment]
Makes sense that the correct python usage should be floor division (`//`).
